ansible/playbooks/tasks/aws-cluster-hana.yaml

```diff
@@ -22,44 +23,104 @@
   changed_when: false
 
 - name: Ensure maintainence mode is active
-  ansible.builtin.command: crm configure property maintenance-mode=true
+  ansible.builtin.command:
+    cmd: crm configure property maintenance-mode=true
   when: crm_maintainence_mode is false or crm_maintainence_mode == 'unknown'
 
 - name: Create HANA topology resource
-  ansible.builtin.command: "crm configure primitive rsc_SAPHanaTopology_HDB_{{ sap_hana_install_sid }}{{ sap_hana_install_instance_number }} ocf:suse:SAPHanaTopology operations $id=\"rsc_sap2_HDB_{{ sap_hana_install_sid }}{{ sap_hana_install_instance_number }}-operations\" op monitor interval=\"10\" timeout=\"600\" op start interval=\"0\" timeout=\"600\" op stop interval=\"0\" timeout=\"300\" params SID=\"{{ sap_hana_install_sid }}\" InstanceNumber=\"{{ sap_hana_install_instance_number }}\""
+  ansible.builtin.command:
+    cmd: |
+      crm configure primitive
+      rsc_SAPHanaTopology_HDB_{{ sap_hana_install_sid }}{{ sap_hana_install_instance_number }}
+      ocf:suse:SAPHanaTopology
+      operations
+      $id="rsc_sap2_HDB_{{ sap_hana_install_sid }}{{ sap_hana_install_instance_number }}-operations"
```

Comment:
Did you confirm this `$id` is not replaced by anything?

Reply:
yes confirmed by adding a `crm configure show` at the end